recipe/meta.yaml

```diff
@@ -91,24 +116,9 @@ test:
   commands:
     - pip check
     - cd tests
-      # TODO: determine why OpenMP on Windows build workers is unreliable
+    # TODO: determine why OpenMP on Windows build workers is unreliable
     - set SKIP_OMP=true                # [win]
-    # https://github.com/conda-forge/imagecodecs-feedstock/pull/32#issuecomment-863041029
-
-    # disable test_jpeg2k_numthreads on all platform except win: it causes segmentation fault.
-    
-    # disable test_zfp as they are all failing on osx-64
-    - pytest -vv test_imagecodecs.py -k "not (test_jpeg_rgb_mode or test_jpeg2k_numthreads or test_image_roundtrips[jpeg2k-uint16-rgb-out-bytearray-None] or test_zfp)"  # [osx and x86_64]
-    
-    - pytest -vv test_imagecodecs.py -k "not (test_jpeg_rgb_mode or test_jpeg2k_numthreads or test_image_roundtrips[jpeg2k-uint16-gray or test_image_roundtrips[jpeg2k-uint16-graya or test_image_roundtrips[jpeg2k-uint16-rgba or test_image_roundtrips[jpeg2k-uint16-rgb)"  # [osx and arm64]
-    - pytest -vv test_imagecodecs.py -k "not (test_jpeg_rgb_mode or test_jpeg2k_numthreads or test_image_roundtrips[jpeg2k-uint16-rgb-out-bytearray-None)"  # [linux and not s390x]
-    - pytest -vv test_imagecodecs.py -k "not (test_jpeg2k_numthreads or test_packints_decode or test_floatpred or test_float24 or test_jpegsof3 or test_image_roundtrip or test_spng_encode or test_ljpeg or test_jpeg_rgb_mode or test_cms_format)"  # [s390x]
-
-    - pytest -vv test_imagecodecs.py -k "not (test_cms_profile or test_jpeg_rgb_mode)"  # [win]
-
-    # CAPTURING FAILING S390x tests to be reviewed later (These tests are failing most likely from an upstream Endianess issue)
-    # These may be fixed with future versions but needs to be checked.
-    # - pytest -vv test_imagecodecs.py -k "test_jpeg2k_numthreads and test_packints_decode and test_floatpred and test_float24 and test_jpegsof3 and test_image_roundtrips and test_spng_encode and test_ljpeg and test_jpeg_rgb_mode and test_cms_format" || true  # [s390x]
+    - pytest -vv test_imagecodecs.py -k "not ({{ tests_to_skip }})"
```

Comment:
do you want to include the expected_codecs test from conda-forge? It could be a way to clearly document what's supported. maybe at the top of the feedstock.

Reply:
Yes, thanks for reminding me.

Reply:
I've checked that there are ~200 codecs at all, see https://github.com/cgohlke/imagecodecs/blob/v2024.9.22/tests/test_imagecodecs.py. If the codec is missing, then the corresponding test is skipped. I'll look at the logs and see which codecs are skipped